.eslintignore

```diff
@@ -1,3 +1,4 @@
+lib/internal/v8.js
```

Comment:
You should be able add comments by starting lines with `#` – if you comment here in the PR, that sounds like a good indicator that it makes sense to do so. :)